src/OrchardCore.Modules/OrchardCore.ContentTypes/Views/ContentTypeSettings.Edit.cshtml

```diff
@@ -66,13 +66,12 @@
 <div class="mb-3">
     <div class="w-md-75 w-xl-50">
         <label asp-for="Stereotype">@T["Stereotype"]</label>
-        <input asp-for="Stereotype" list="stereotypeOptions" type="text" class="form-control">
-        <datalist id="stereotypeOptions">
+		<select asp-for="Stereotype" id="stereotypeOptions" class="form-select">
```

Comment:
Please revert this change because stereotypes could be editable

Reply:
I reverted what you asked for, but it still looks like it has been edited because it messed up the spacing. 
But just I was thinking should the stereotypes be editable though? What are your thoughts @Piedone?

Reply:
I might be wrong here, so I will wait for @MikeAlhayek